olefile/olefile.py

```diff
@@ -1022,6 +1022,7 @@ def __init__(self, filename=None, raise_defects=DEFECT_FATAL,
             Unicode by default on Python 3+, UTF-8 on Python 2.x.
             (new in olefile 0.42, was hardcoded to Latin-1 until olefile v0.41)
         """
+        self.filename = filename
```

Comment:
I think `self.filename` is unused and can be dropped.

Reply:
You're right. I've removed it 👍